docs/source/guides/logical_types_and_semantic_tags.ipynb

```diff
@@ -0,0 +1,664 @@
+    "    'ip_addresses': [\"172.16.254.1\",\n",
+    "                     \"192.0.0.0\",\n",
+    "                     \"2001:0db8:0000:0000:0000:ff00:0042:8329\",\n",
+    "                     \"192.0.0.0\"],\n",
+    "})\n",
+    "strings_df.ww.init(logical_types={\n",
+    "    'natural_language':'NaturalLanguage',\n",
+    "    'addresses':'Address',\n",
+    "    'filepaths':'FilePath',\n",
+    "    'full_names':'PersonFullName',\n",
+    "    'phone_numbers':'PhoneNumber',\n",
+    "    'urls':'URL',\n",
+    "    'ip_addresses':'IPAddress',\n",
+    "})\n",
+    "strings_df.ww"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "secure-delta",
+   "metadata": {},
+   "source": [
+    "## ColumnSchema objects\n",
+    "\n",
+    "Now that we've gone in-depth on semantic tags and logical types, we can start to understand how they're used together to build Woodwork tables and define type spaces.\n",
+    "\n",
+    "A `ColumnSchema` is the typing information for a single column. We can obtain a `ColumnSchema` from a Woodwork-initialized DataFrame as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "adult-portfolio",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Woodwork typing info for a DataFrame\n",
+    "retail_df = ww.demo.load_retail()\n",
+    "retail_df.ww"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "separate-slovenia",
+   "metadata": {},
+   "source": [
+    "Above is the typing information for a Woodwork DataFrame. If we want, we can access just the schema of typing information outside of the context of the actual data in the DataFrame."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "quiet-virus",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# A Woodwork TableSchema\n",
+    "retail_df.ww.schema"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "opened-payment",
+   "metadata": {},
+   "source": [
+    "The representation of the `woodwork.table_schema.TableSchema` is only different in that it does not have a column for the physical types.\n",
+    "\n",
+    "This lack of a physical type is due to the fact that a `TableSchema` has no data, and therefore no physical representation of the data. We often rely on physical typing information to know the exact pandas or Dask or Koalas operations that are valid for a DataFrame, but for a schema of typing information that is not tied to data, those operations are not relevant.\n",
+    "\n",
+    "Now, let's look at a single column of typing information, or a `woodwork.column_schema.ColumnSchema` that we can aquire in much the same way as we can select a Series from the DataFrame: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "southern-comedy",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Woodwork typing infor for a Series\n",
+    "quantity = retail_df.ww['quantity']\n",
+    "quantity.ww"
```

Comment:
not sure if this is intentional, but the output gets cuttoff

Reply:
played around a bit with trying to change the scrolling behavior by toggling `toggle all cells output scrolled` (it's the `scrolled` entry in the metadata). It didn't seem to have any impact on the outputs in the docs (or in the notebook itself, which wraps for this cell's outputs).

Unless there's some config we can change in the docs, the best bet might be changing the repr to put each element of the typing info on a separate line. Not sure if the change would be worth it, though.

Reply:
created an issue to handle https://github.com/alteryx/woodwork/issues/1092